### DISCUSSION TITLE
See on the farm which boosts are active

### DISCUSSION THREAD

**Discussion by ZipManZero** [sunflower-land/sunflower-land#1970] [Ideas]
The idea would be to have a session on the game where players could see which boosters they have active.

For example, NFT X +20% crop speed ... NTF Y Stone 10% faster, and so on

A way to visually know which nft are active on the farm, which are accumulating, testing combinations etc.

I believe it would add a lot to the user experience.

thx #152886

**Comment by xSubZero** [comment]
This is a great idea, in the previous version of the game (V2) we had an overview of each person's nfts when visiting the land, now we don't get this anymore.

**Comment by AzaroQL** [comment]
A button 'Active Boosts' with a table of all active boosts could be a way to go
![image](https://user-images.githubusercontent.com/9656961/211226845-0ee6c16d-b6d3-456e-8a30-26e12ebc7e3c.png)

**Reply by cryptosmikey** [reply]
Excellent idea
